extensions.json

```
@@ -208,6 +208,22 @@
       "author": "Kapdap",
       "url": "https://github.com/Kapdap/freshrss-extensions",
       "type": "gh-subdirectory"
+    },
+    {
+      "name": "TinyTinyRSS API",
+      "description": "Provides an API compliant with TinyTinyRSS applications.",
+      "version": 0.2,
+      "author": "Marien Fressinaud",
+      "url": "https://github.com/FreshRSS/Extensions",
+      "type": "gh-subdirectory"
+    },
+    {
+      "name": "Title-Wrap",
+      "description": "Applies a line-wrap to long article titles, as opposed to truncating the title when it overflows its display area.",
+      "version": 0.2,
+      "author": "₣rans de Jonge, Joris Kinable",
```

I don't mind the extra flourish, but how'd you even type that? :-P
```suggestion
      "author": "Frans de Jonge, Joris Kinable",
```

---

Fixed